build.sbt

```diff
@@ -18,16 +18,16 @@ import sbt.{Project, addCompilerPlugin, _}
 import sbt.librarymanagement.CrossVersion
 import com.github.sbt.git.SbtGit.GitKeys._
 
-// Variables:
-val scioVersion = "0.13.6"
-val beamVersion = "2.52.0" // keep in sync with scio
+val scioVersion = "0.14.0"
+
+val beamVersion = "2.53.0" // keep in sync with scio
 val avroVersion = "1.8.2" // keep in sync with scio
 val scalacheckShapelessVersion = "1.3.1"
 val scalatestVersion = "3.2.17"
 val scalatestMockitoVersion = "3.1.0.0"
 val jodaTimeVersion = "2.10.10" // keep in sync with scio
-val magnoliaVersion = "1.1.3" // keep in sync with scio
-val ratatoolVersion = "0.4.3" // keep scio versions in sync
+val magnoliaVersion = "1.1.8" // keep in sync with scio
+val ratatoolVersion = "0.4.4"
```

Comment:
Updated to 0.4.4 in anticipation of Ratatool being released! Once it's released we should only have to retrigger the CircleCI build.